Algorithm/test/headerstack.cxx

```diff
@@ -41,7 +41,7 @@ BOOST_AUTO_TEST_CASE(test_headerstack)
   dh.payloadSize = 0;
 
   using Name8Header = o2::header::NameHeader<8>;
-  Name8Header nh("NAMEDHDR");
+  Name8Header nh("NAMDHDR");
```

Comment:
@ktf I think the warning is actually pointing here where a string of the same length as the NameHeader string length is used, so there is no space for zero termination, and this is what the warning says.

Reply:
yes, agreed. Maybe we should throw if the Name8Header string has length > 7...

Reply:
or assert...

Reply:
Then I would prefer to throw, since we build with NODEBUG

Reply:
I have changed the type of the argument so that we can use static_assert